File: iniciante/C++/1065-Even_between_5_numbers.cpp
#include <iostream>
 
using namespace std;
 
int main() {
 
    int i, num, contPar;
    for (i=0;i<5;i++) {
        cin >> num;
        if (num%2==0) {
            contPar+=1;
        }
    }
    cout << contPar << " valores pares\n";
    return 0;
}

File: iniciante/Dart/1065-Even_between_5_numbers.dart
import 'dart:io';

void main() {
  int even = 0;
  for (int i = 0; i < 5; i++) {
    // Remove the ! before submitting the solution...
    var num = int.parse(stdin.readLineSync()!);
    if (num % 2 == 0) {
      even += 1;
    }
  }
  print("${even} valores pares");
}
